contrib/charts/cert-manager/templates/deployment.yaml

```diff
@@ -40,6 +40,10 @@ spec:
         {{- if .Values.ingressShim.extraArgs }}
           args:
 {{ toYaml .Values.ingressShim.extraArgs | indent 10 }}
+        {{- else }}
+          args:
```

Comment:
Don't need the second 'args' - move the original one to before the `if` 😄

Reply:
I also think we should set these fields *iff* their respective values are set in the values.yaml, and *not* just if extraArgs isn't specified. For example, a user might specify `--set ingressShim.extraArgs={--v=4} --set ingressShim.defaultIssuerName=letsencrypt-prod --ingressShim.defaultIssuerKind=ClusterIssuer`, which should ultimately result in the args:

`--v=4 --default-issuer-name=letsencrypt-prod --default-issuer-kind=ClusterIssuer`

If a user *has* specified `letsencryptEmail` and **has not** manually specified `ingressShim.defaultIssuerName`, we **should** automatically set the default issuer name to `letsencrypt-staging` for them. The same goes for issuer kind in this case 😄